src/main/kotlin/org/wycliffeassociates/otter/common/persistence/IDirectoryProvider.kt

```diff
@@ -12,4 +12,5 @@ interface IDirectoryProvider {
 
     val userProfileImageDirectory: File
     val userProfileAudioDirectory: File
+    val audioPluginsDirectory: File
```

Comment:
I think this can be kept singular (plugin directory), especially since the directory is the only use of it in the plural sense. (this change affects the above two files as well)